src/codegen/Common.ts

```diff
@@ -128,7 +128,7 @@ const common = {
 
   // Data model bindings:
   IDataModelBindingsSimple: () =>
-    new CG.obj(new CG.prop('simpleBinding', new CG.str()))
+    new CG.obj(new CG.prop('simpleBinding', new CG.str()), new CG.prop('label', new CG.str().optional()))
```

Comment:
Not all components with a simple binding should have the option to use a `label` binding (for example, it does not make sense for an `Input` component), so this should just be set on the `IDataModelBindingsOptionsSimple` type.